notebooks/samples/explanations/ai-explanations-tabnet-algorithm.ipynb

```diff
@@ -0,0 +1,540 @@
+{
```

Comment:
<p>Can you move these sentences down into the "Overview" section below? </p><p> </p><p>Also, rewrite as:</p><p> </p><p>AI Platform provides a built-in algorithm based on [TabNet](https://arxiv.org/abs/1908.07442).</p><p>Learn how to [train and deploy a model with the TabNet built-in algorithm](https://cloud.google.com/ai-platform/training/docs/algorithms/tab-net-start).</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/182/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>